app/lib/frontend/templates/views/pkg/package_list.dart

```diff
@@ -189,12 +200,38 @@ d.Node _item({
           if (labeledScoresNode != null) labeledScoresNode,
         ],
       ), // end of packages-header
-
-      d.p(classes: ['packages-description'], text: description),
-      d.p(classes: ['packages-metadata'], child: metadataNode),
-      if (tagsNode != null) d.div(child: tagsNode),
-      if (apiPages != null && apiPages.isNotEmpty)
-        d.div(classes: ['packages-api'], child: _apiPages(apiPages)),
+      d.div(classes: [
+        'packages-container'
+      ], children: [
+        d.div(
+          classes: ['packages-body'],
+          children: [
+            d.div(classes: ['packages-description'], text: description),
+            d.p(classes: ['packages-metadata'], child: metadataNode),
+            if (tagsNode != null) d.div(child: tagsNode),
+            if (apiPages != null && apiPages.isNotEmpty)
+              d.div(classes: ['packages-api'], child: _apiPages(apiPages)),
+          ],
+        ),
+        if (thumbnailUrl != null && requestContext.isExperimental)
+          d.div(classes: [
+            'screenshot-thumbnail'
+          ], children: [
+            d.img(
+                image: d.Image(
+                    alt: 'screenshot',
+                    width: null,
+                    height: null,
+                    src: thumbnailUrl)),
+            d.img(
+                classes: ['collections-icon'],
+                image: d.Image(
+                    height: null, // We handle sizing in css
```

Comment:
I think the lighthouse analysis will downscore us if we don't put width and height here. Let's just repeat them here too, event though the CSS may have them as well.